README.md

```diff
@@ -1,15 +1,40 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/FgMJElkj)
 # Theory vs. Practice
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  
+1. Different machines have different kinds of computational mechanisms. Its possible to get different results running the same program on two different machines with distinct capabilities.
```

Comment:
What do you mean by "computational mechanism"? And if you're getting different results on different machines, there's probably something wrong...